selectors.yml

```diff
@@ -130,21 +187,57 @@ selectors:
   10:
     selector: 3734403246176062136
     name: "ethereum-mainnet-optimism-1"
+  25:
+    selector: 1456215246176062136 
+    name: "cronos-mainnet"
+  40:
+    selector: 1477345371608778000
+    name: "telos-evm-mainnet"
+  46:
+    selector: 8866418665544333000
+    name: "polkadot-mainnet-darwinia"
+  52:
+    selector: 1761333065194157300
+    name: "coinex_smart_chain-mainnet"
   56:
     selector: 11344663589394136015
     name: "binance_smart_chain-mainnet"
   100:
     selector: 465200170687744372
     name: "gnosis_chain-mainnet"
+  106: 
+    selector: 374210358663784372 
+    name: "velas-mainnet"
   137:
     selector: 4051577828743386545
     name: "polygon-mainnet"
+  196:
+    selector: 3016212468291539606
+    name: "ethereum-mainnet-xlayer-1"
   199:
     selector: 3776006016387883143
     name: "bittorrent_chain-mainnet"
+  252:
+    selector: 1462016016387883143 
+    name: "fraxtal-mainnet"
   255:
     selector: 3719320017875267166
     name: "ethereum-mainnet-kroma-1"
+  259:
+    selector: 8239338020728974000
+    name: "neonlink-mainnet"
+  324: 
+    selector: 1562403441176082196
+    name: "ethereum-mainnet-zksync-1" 
+  397:
+    selector: 2039744413822257700
+    name: "near-mainnet"
+  463:
+    selector: 1939936305787790600
+    name: "areon-mainnet"
+  592: 
+    selector: 6422105447186081193 
+    name: "polkadot-mainnet-astar"
```

Comment:
What? They have multiple mainnets?

Reply:
Those are 2 different chains. One is zkEVM, the other isn't